utils/elect-rc-leader

```diff
@@ -47,6 +47,14 @@ get_leader_node() {
         jq -r '.[] | .Node'
 }
 
+get_node_name() {
+    # hare/libexec is added to PATH env variable only by hctl 'grounding'
+    # script.
+    # In non-hctl context, the scripts should use absolute path instead
+    # when accessing scripts under hare/libexec/.
+    /opt/seagate/cortx/hare/libexec/node-name
```

Comment:
Instead of writing full path every time, can we have some macro name for this path, like we have
CORTX_HA_BIN
HARE_BASE_DIR
etc.

Reply:
@suvratjoshi where would you propose to put those macros to?

Reply:
I am not sure on that how we follow this, so not sure where to put it.
Do we have a common file where we can have such macros and can be accessible globally? (Like what we do in a C language code)

Reply:
@suvratjoshi , either its a function or a macro. I think function is good.